lib/tasks/grade.rake

```diff
@@ -92,6 +97,40 @@ namespace :grade do
 
 end
 
+def sync_specs_with_source
+  base_url = "https://api.github.com/repos/"
```

Comment:
Would the [octokit gem](https://octokit.github.io/octokit.rb/Octokit/Client/Contents.html#contents-instance_method) come in handy here?